.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+---
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    labels:
+      - automation
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-major"]
```

Comment:
Should we also group all otel pacakges to prevent too many PRs when otel version up happens?

```yaml
    groups:
      otel:
        patterns:
        - "go.opentelemetry.io/*"
```

Reply:
Good idea! Added in https://github.com/elastic/opentelemetry-collector-components/pull/12/commits/581b12076830010e52e9c8ccafd882e52d8b2e64